app/javascript/controllers/textarea_autogrow_controller.js

```diff
@@ -23,7 +23,11 @@ export default class extends Controller {
 
   debouncedAutogrow = debounce(() => this.autogrow(), 50)
   autogrow() {
-    this.element.style.height = 'auto'
-    this.element.style.height = `${this.element.scrollHeight + 2}px` // the +2 is a hack to make the size not jump on load. The scrollHeight differs from than actual height for the empty state
+    const newHeight = `${this.element.scrollHeight + 2}px` // the +2 is a hack to make the size not jump on load. The scrollHeight differs from than actual height for the empty state
+    if (this.element.style.height && this.element.style.height != newHeight) {
+      this.element.style.height = 'auto'
+      this.element.style.height = newHeight
+      window.dispatchEvent(new Event('resize'))
+    }
```

Comment:
When I added this dispatchEvent I discovered this style.height was being set a lot more frequently than it needed to be so this extra logic only sets it when it changed